depends/packages/qt.mk

```diff
@@ -118,65 +118,91 @@ define $(package)_extract_cmds
   echo "$($(package)_qttools_sha256_hash)  $($(package)_source_dir)/$($(package)_qttools_file_name)" >> $($(package)_extract_dir)/.$($(package)_file_name).hash && \
   $(build_SHA256SUM) -c $($(package)_extract_dir)/.$($(package)_file_name).hash && \
   mkdir qtbase && \
-  tar --strip-components=1 -xf $($(package)_source) -C qtbase && \
+  tar --no-same-owner --strip-components=1 -xf $($(package)_source) -C qtbase && \
```

Comment:
why is that --no-same-owner flag necessary?

Reply:
Honestly I just copied the whole thing over from Bitcoin then tried to clean it up, then realised it wasn't going to be a small task where I could just give you the next steps.

It comes from https://github.com/bitcoin/bitcoin/commit/89bee1bdbf7f77639e053dd004629e0db0e10724 but I'm happy you ignore this (or any of the other changes in the PR), as long as the result works :)